packages/api-gateway/src/drivers/http/routes/admin_panel/api/schema.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+//TODO_JAIRO: schema doesn't work propertly
+const changeUSerStatusSchema = {
+	description: 'change user status: Given a status and a reason, change the status of a user and send a notification to the user',
+	tags: ['AdminPanel'],
```

Comment:
The tag for admin panel is `Admininistration panel`, can you replace it?
```suggestion
	tags: ['Admininistration panel'],
```